Products/SQLAlchemyDA/da.py

```diff
@@ -39,6 +39,44 @@
 }
 
 
+# Global registry of named SQLAlchemyDA instances
+_da_registry = {}
+
+
+def register_da(name, da_instance):
+    """
+    Register an SQLAlchemy database adapter by name as part of a module
+    level dict. This might be called early in Zope startup, so this type
+    of registration is necessary instead of a zope.component registration.
+    (The same reason z3c.sqlalchemy uses a module dict for registration)
+    """
+    global _da_registry
```

Comment:
I've had a long habit of doing this, just for the sake of making it more obvious, to hang a big red warning flag that we're dealing with a module level global. I guess in this case though, the global is declared so closeby that it doesn't really add any clarity. I'll remove it.